zipline/algorithm.py

```diff
@@ -1253,6 +1254,8 @@ def future_chain(self, root_symbol, as_of_date=None):
             Date at which the chain determination is rooted. I.e. the
             existing contract whose notice date is first after this date is
             the primary contract, etc.
```

Comment:
This should also spell out explicitly that the default behavior is the current simulation session.